search-index-import-util/src/main/java/no/unit/nva/search/ImportDataRequest.java

```diff
@@ -0,0 +1,61 @@
+package no.unit.nva.search;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.net.URI;
+import java.util.Objects;
+import java.util.Optional;
+import nva.commons.core.JacocoGenerated;
+import nva.commons.core.JsonSerializable;
+
+public class ImportDataRequest implements JsonSerializable {
+
+    public static final String S3_LOCATION_FIELD = "s3Location";
+    @JsonProperty(S3_LOCATION_FIELD)
+    private final URI s3Location;
+
+    @JsonCreator
+    public ImportDataRequest(@JsonProperty(S3_LOCATION_FIELD) String s3Location) {
```

Comment:
This is the input of the function. A location with a DynamoDb export in ION format.